SableCC/Generated/GOATCode.sablecc

```diff
@@ -0,0 +1,414 @@
+Package GOATCode;
+
+/*
+    This file describes the grammar accepted in our language.
+    Helpers and tokens can be seen below, which are used to identify the accepted words of the language.
+    
+    Productions for the concrete syntax tree can be seen below the tokens in the production section.
+    At the bottom of the file the productions for the AST transformations can be found.
+
+    The mapping from the CST to the AST happens in the curly brackets with the arrow {-> ...}
+    If there is the 'New' keyword in the mapping it generates a new node in the AST 
+    and everything in the () is passed to the AST productions.
+*/
+
+Helpers
+  all = [0 .. 127];
+  tab   = 9;
+  cr    = 13;
+  lf    = 10;
+  digit = ['0'..'9'];
+  lowercase = ['a'..'z'];
+  uppercase = ['A'..'Z'];
+  letter  = lowercase | uppercase;
+  not_curleys = [all - ['{'+'}']];
+  idletter = letter | '_';
+  idchar  = letter | '_' | digit;
+  xyz = 'x' | 'y' | 'z';
+  star = '*';
+  not_star = [all - star];
+  not_star_slash = [not_star - '/'];
+  eol_char = cr | lf | cr lf;
+  not_eol_char = [all - [cr + lf]];
+  spacing = eol_char | ' ' | tab;
+
+Tokens
+  eol   = cr | lf | cr lf;
+  blank = ' ' | tab;
+  assignment_op = '=';
+  multiplication_op  = '*';
+  modulo_op  = '%';
+  division_op = '/';
+  plus_op  = '+';
+  minus_op = '-';
+  equal = '==';
+  less_than = '<';
+  less_than_or_equal = '<=';
+  greater_than = '>';
+  greater_than_or_equal = '>=';
+  not = '!';
+  not_equal = '!=';
+  l_par = '(';
+  r_par = ')';
+  l_cur = '{';
+  r_cur = '}';
+  comma = ',';
+  vector = 'vector';
+  int = 'int';
+  const = 'const';
+  float = 'float';
+  bool = 'bool';
+  void = 'void';
+  bool_value = 'true' | 'false';
+  if = 'if';
+  else = 'else';
+  repeat = 'repeat';
+  while = 'while';
+  return = 'return';
+  build = 'build';
+  walk = 'walk';
+  gcode_literal = 'gcode' spacing* '{' not_curleys* '}';
```

Comment:
nitpitck but this means that the gcode is the only thing that allows multiple end of lines before a block